parts/cleanup.sh

```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 set -eux -o pipefail
 
-systemctl disable dhcpcd
 systemctl disable hciuart
 
-apt-get remove --yes build-essential devscripts
```

Comment:
What's the reason for keeping `build-essential` on the device? There shouldn't be much to build once the image has built?

Reply:
When installing things from pip that have c-bindings you tend to need this or you get strange errors while installing. Generally it's useful and not massive.

Reply:
It's a meta package with a lot in it. We shouldn't need to be installing things after build, right?

Reply:
It's only 45MB and gives us flexibility,